.proxyrc.json

```diff
@@ -6,7 +6,7 @@
     "target": "http://127.0.0.1:8002/"
   },
   "/processes/": {
-    "target": "http://127.0.0.1:8002/"
+    "target": "http://127.0.0.1:8001/"
```

Comment:
Just to be sure I'm getting this right, `8001` is the expressjs port and `8002` the django one, and we now serve the processes endpoint from express instead of django?

Reply:
That's it :+1: